src/renderer/models/types/networkMetrics.ts

```diff
@@ -0,0 +1,20 @@
+import {ContainerType, ListType, NumberUintType} from "@chainsafe/ssz";
+import {StringType} from "./basic";
+import {NetworkMetric, NetworkMetrics} from "../networkMetrics";
+
+const MILLION = 1000000;
+
+export const NetworkMetricType = new ContainerType<NetworkMetric>({
+    fields: {
+        url: new StringType(),
+        code: new NumberUintType({byteLength: 32}),
+        latency: new NumberUintType({byteLength: 32}),
+        time: new NumberUintType({byteLength: 32}),
+    },
+});
+
+export const NetworkMetricsType = new ContainerType<NetworkMetrics>({
+    fields: {
+        records: new ListType({elementType: NetworkMetricType, limit: MILLION}),
```

Comment:
probably better to create unboundedArray, with serialize and json methods implemented however you wan't. Milion might not be enough.

Reply:
I wanted to bu limit is a required field, that why I put a million hoping it will be enoight

Reply:
I've answered this issue in another PR